sku/client.go

```diff
@@ -125,6 +125,10 @@ func (c Client) Update(id string, params *stripe.SKUParams) (*stripe.SKU, error)
 			body.Add("image", params.Image)
 		}
 
+		for k, v := range params.Attrs {
+			body.Add(fmt.Sprintf("attributes[%v]", k), v)
```

Comment:
I worry quite a bit about this sort of interface:

1. If you don't understand that the update will act as a replacement, then you may end up overwriting your old attributes because you didn't explicitly copy them over.
2. If you do understand that the update will act as a replacement, you won't succeed in overwriting your old attributes if your new set is smaller than your old set (i.e. because attributes at the upper indexes will still exist). Is it even possible to blank this field out like you can with a product?